### ISSUE TITLE
Invalid element tooltips are not screen reader accessible

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#6309]
Invalid element tooltips are tied to the input element visually, and this works great for people that can see the visual tie-in.

The visual tie does not work well on screen readers. @MarcoZehe suggests we add an id attribute to the tooltip, then add an `aria-described-by` attribute to the invalid input element.
